server/controllers/prisonerCellHistoryController.ts

```diff
@@ -7,9 +7,12 @@ import { formatDate, formatDateTime, formatDateTimeISO } from '../utils/dateHelp
 import { NameFormatStyle } from '../data/enums/nameFormatStyle'
```

Comment:
Changes for this page not really related but interaction via one of the helpers showed it to not be quite correct in terms of typings etc